README.md

```diff
@@ -1,14 +1,31 @@
 # My React dApp
 
-## Install dependencies
+This monorepo was designed to provide a developer-friendly experience to Alephium ecosystem newcomers. It is split into 2 parts:
+
+- app: contains the React frontend part of the dApp
+- contracts: contains the dApp contracts
+
+It uses **yarn workspaces** to manage both app and contract projects from the monorepo root.
+
+To get started quickly, follow these steps:
+
+## Set up a devnet
+
+Start a local devnet for testing and development. Please refer to the [Getting Started documentation](https://docs.alephium.org/full-node/getting-started#devnet).
+
+## Install an Alephium wallet
+
+Download an [Alephium wallet](https://alephium.org/#wallets), connect to it to your devnet and import this mnemonic to have 4,000,000 token allocated for your addresses:
 
 ```
-yarn install
+vault alarm sad mass witness property virus style good flower rice alpha viable evidence run glare pretty scout evil judge enroll refuse another lava
```

Comment:
I would recommend devs to request 1000$ALPH for devnet using the faucet so it's consistent with testnet.

doc: https://docs.alephium.org/wallet/extension-wallet/getting-started#alph-token-faucet

<img width="272" alt="Screenshot 2024-10-31 at 15 49 52" src="https://github.com/user-attachments/assets/dac123e3-0b2c-4f3a-9e56-65aed27e2d6c">

Reply:
Ah, ok. The DW only provides a testnet faucet and the mobile wallet none. So I thought restoring this mnemonic would work in every wallet the devs choose to use.